src/App.js

```diff
@@ -22,15 +22,26 @@ class App extends React.Component {
 		};
 	}
 
-	//handleclick
-
-	//handleChange
 	handleChange = event => {
 		this.setState({
 			controlledText: event.target.value
 		});
 	};
 
+	handleNew = event => {
+		event.preventDefault();
+		let newtodos = this.state.todos.slice();
+		newtodos.push({
+			task: this.state.controlledText,
+			id: Date.now(),
```

Comment:
nice use of the Date.now() remember that in production code you can use the uuid module from npm